app/components/FileLoader.js

```diff
@@ -22,7 +22,6 @@ import Scroller from './common/Scroller';
 
 const GAME_BATCH_SIZE = 50;
 const MIN_GAME_LENGTH_SECONDS = 30;
```

Comment:
What do you think about having this value returned from the `fileLoader`? Otherwise there will be two sources of truth if we ever change that min game length value.